SYCL/ESIMD/Stencil.cpp

```diff
@@ -79,7 +72,13 @@ bool CheckResults(float *out, float *in) {
   return true;
 }
 
-int main(void) {
+int main(int argc, char *argv[]) {
+  if (argc != 2) {
+    std::cerr << "Usage: stencil.exe <dim_size>" << std::endl;
+    exit(1);
+  }
+  const unsigned DIM_SIZE = atoi(argv[1]);
+  const unsigned SQUARE_SZ = DIM_SIZE * DIM_SIZE + 16;
```

Comment:
looks like code duplication. Can we move common part to header?

Reply:
Will do that as a separate patch.